src/avkit.cs

```diff
@@ -309,6 +243,48 @@ interface AVPlayerViewControllerDelegate
 		[NoiOS, TV (11,0), NoWatch, NoMac]
 		[Export ("playerViewController:willTransitionToVisibilityOfTransportBar:withAnimationCoordinator:")]
 		void WillTransitionToVisibilityOfTransportBar ([NullAllowed] AVPlayerViewController playerViewController, bool visible, [NullAllowed] IAVPlayerViewControllerAnimationCoordinator coordinator);
+		
+		[iOS (12,0), NoTV, NoWatch, NoMac]
+		[Export ("playerViewController:willBeginFullScreenPresentationWithAnimationCoordinator:")]
+		void WillBeginFullScreenPresentation (AVPlayerViewController playerViewController, UIViewControllerTransitionCoordinator coordinator);
+		
+		[iOS (12,0), NoTV, NoWatch, NoMac]
+		[Export ("playerViewController:willEndFullScreenPresentationWithAnimationCoordinator:")]
+		void WillEndFullScreenPresentation (AVPlayerViewController playerViewController, UIViewControllerTransitionCoordinator coordinator);
```

Comment:
Here is the error about UIViewControllerTransitionCoordinator not being in UIKit (despite the fact it is).